Public/Src/Engine/Dll/EngineSchedule.cs

```diff
@@ -1659,7 +1685,7 @@ public SchedulerPerformanceInfo LogStats(LoggingContext loggingContext)
                         pipTwoPhaseCache: pipTwoPhaseCache,
                         symlinkDefinitions: await symlinkDefinitionsTask,
                         buildEngineFingerprint: buildEngineFingerprint,
-                        vmInitializer: VmInitializer.CreateFromEngine(newConfiguration.Layout.BuildEngineDirectory.ToString(pathTable)));
+                        vmInitializer: VmInitializer.CreateFromEngine(newConfiguration.Layout.BuildEngineDirectory.ToString(pathTable), GetLowPrivilegeBuildAccount(), GetLowPrivilegeBuildPassword()));

```

Comment:
>CreateFromEngine [](start = 53, length = 16)

It is unclear when the secret is released so it can be cleaned up.
Should we have some explicit disposing of the secret from memory.

My concern is that the secret is longer lived in memory than needed and if anything is a dump or we accidentally expose a buffer somewhere to users we have a higher chance to leak the secret.

Reply:
True, I will change it to securitystring.